fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/navbar/NavigationBar.kt

```diff
@@ -74,16 +76,22 @@ fun BrowserNavBar(
             browserState.normalTabs.size
         }
     }.value
+    val canGoBack = browserStore
+        .observeAsComposableState { state -> state.selectedTab?.content?.canGoBack }.value ?: false
```

Comment:
Can use `observeAsState` as above that returns a non null result. Same for the one below.

An idea 💡 : Could create a `NavBarUIModel`/State to drive the NavBars so there's only one `observeAsState` that maps to the same `UIModel`.

Reply:
adjusted to `observeAsState`👍

I like the idea but couldn't find any examples of the usage, usually it's multiple of `observeAsComposableState` or `observeAsState`, and hesitant to introduce a new approach in this PR (trying to merge stuff before Friday). Could you point me to an example? Could do it as a follow up.